spec/components_spec.rb

```diff
@@ -355,8 +355,8 @@
 
 RSpec.describe "raw" do
   it 'creates a wrapper that ignores anything inside' do
-    input = "<raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw>"
-    expected = "<<LCG Program\TG LCG Coupon Code Default='246996'>>"
+    input = "<body><raw><<LCG Program\TG LCG Coupon Code Default='246996'>></raw></body>"
+    expected = "<?xml version=\"1.0\"?>\n<body><<LCG ProgramTG LCG Coupon Code Default='246996'>></body>\n"
```

Comment:
Ah interesting.  I believe this is related to https://github.com/zurb/inky-rb/issues/11.  I think we actually should make Inky-rb smart enough auto-insert a root node if one doesn't exist.